docs/decisions/0002-multiple-exam-systems.rst

```diff
@@ -0,0 +1,56 @@
+Support for Multiple Exam Systems
+=================================
+
+Status
+------
+
+Propsed (circa May 2022)
+
+Context
+-------
+The addition of this service to Open edX will result in two separate services
+for managing exams, this IDA and the `edx-proctoring`_ plugin. Because this IDA will
+not support existing vendor integrations with the same feature set, edx-proctoring cannot be deprecated
+right away. As a result, we need a solution that allows course teams to choose a proctoring tool
+that is set up in either service but avoids adding configuration of lots of switching code to edx-platform.
+
+Decision
+--------
+This service will include all logic necessary to determine which proctoring implementation a particular API
+request is relevant for and forward requests `edx-proctoring`_ where appropriate.
```

Comment:
requests to 'edx-proctoring'

Reply:
Also terminology nit - forward also is a bit of a weird word because it sounds like it's going to another process somewhere, but we'll just have edx-proctoring lib embedded in the exam service right?

maybe "delegate" rather than "forward" ?

Reply:
well we can't actually embed it b/c that would involve pulling the existing database over and rebuilding the entire edx-proctoring python API with rest endpoints. The proposal here would only require rest endpoints for the intersection of calls both edx-exams and edx-proctoring need.

Reply:
ahhhh I see carry on :)